en/manual/engine/entity-component-model/usage.md

```diff
@@ -0,0 +1,91 @@
+# ECS Usage
+
+## Classes
+
+The three parts of "Entity Component System" map to the following classes:
+
+- Entity - [`Stride.Engine.Entity`](https://doc.stride3d.net/latest/en/api/Stride.Engine.Entity.html)
+- Component - [`Stride.Engine.EntityComponent`](https://doc.stride3d.net/latest/en/api/Stride.Engine.EntityComponent.html)
+- System - [`Stride.Engine.EntityProcessor`](https://doc.stride3d.net/latest/en/api/Stride.Engine.EntityProcessor.html)
+
+
+## Minimal Setup
+
+A component can be defined by deriving a class from `EntityComponent`.
+
+By adding the attribute `DefaultEntityComponentProcessor` to an `EntityComponent`,  
+an `EntityProcessor` can be assigned to it. This will automatically set up and run  
+the `EntityProcessor` if the `EntityComponent` is in the scene.
+
+An `EntityComponent` also needs to indicate that it can be serialized  
+by adding the attribute `DataContract` to it.
+
+A system can be defined by deriving a class from `EntityProcessor`. 
+
+
+### Code
+
+#### Component
+```csharp
+[DataContract(nameof(MyComponent))]
+[DefaultEntityComponentProcessor(typeof(MyProcessor))]
+public class MyComponent : EntityComponent
+{
+    public int MyValue { get; set; }
+}
+```
+
+#### System
+```csharp
+public class MyProcessor : EntityProcessor<MyComponent>
+{
+    public override void Update(GameTime time)
+    {
+        foreach (var myComponent in ComponentDatas.Values)
+        {
+            Console.WriteLine($"myComponent with value {myComponent.MyValue} at {time.Total.TotalSeconds}");
+        }
+    }
+}
+```
+
+### Additional Note
+An `EntityComponent` can currently not be drag-dropped onto an entity in Game Studio.  
+t has to be added by selecting an entity, and then clicking the "Add component" button  
```

Comment:
missing an `i` in `it`

Reply:
Ah good find, fixed it. Also fixed another type where it was "ill" instead of "will".

Also, good that you say it, because it also made me remember that I forgot another pretty amazing part, which I added now.